src/code/z_demo.c

```diff
@@ -2230,7 +2230,11 @@ void CutsceneHandler_RunScript(PlayState* play, CutsceneContext* csCtx) {
         csCtx->curFrame++;
 
         if (OOT_DEBUG && R_USE_DEBUG_CUTSCENE) {
+#if OOT_DEBUG
+            // Under !OOT_DEBUG this branch is optimized out, but we still need to #if out the reference to
+            // gDebugCutsceneScript
             Cutscene_ProcessScript(play, csCtx, gDebugCutsceneScript);
+#endif
         } else {
             Cutscene_ProcessScript(play, csCtx, play->csCtx.script);
         }
```

Comment:
```suggestion
#if OOT_DEBUG
        if (R_USE_DEBUG_CUTSCENE) {
            Cutscene_ProcessScript(play, csCtx, gDebugCutsceneScript);
        } else {
            Cutscene_ProcessScript(play, csCtx, play->csCtx.script);
        }
#else
        Cutscene_ProcessScript(play, csCtx, play->csCtx.script);
#endif
```
(just so we don't forget, though I don't care that much what the resolution is)